sdk/textanalytics/ai-text-analytics/test/public/utils/recordedClient.ts

```diff
@@ -1,74 +1,54 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { Context } from "mocha";
+import { Test } from "mocha";
 
 import {
+  assertEnvironmentVariable,
   env,
   Recorder,
-  record,
-  RecorderEnvironmentSetup,
-  isLiveMode,
+  RecorderStartOptions,
 } from "@azure-tools/test-recorder";
-import { TokenCredential, ClientSecretCredential } from "@azure/identity";
-import { isNode, createXhrHttpClient } from "@azure/test-utils";
+import { TokenCredential } from "@azure/identity";
+import { createTestCredential } from "@azure-tools/test-credential";
 
 import { AzureKeyCredential, TextAnalyticsClient, TextAnalyticsClientOptions } from "../../../src/";
-import "./env";
 
-const replaceableVariables: { [k: string]: string } = {
+const envSetupForPlayback: { [k: string]: string } = {
   AZURE_CLIENT_ID: "azure_client_id",
   AZURE_CLIENT_SECRET: "azure_client_secret",
   AZURE_TENANT_ID: "88888888-8888-8888-8888-888888888888",
```

Comment:
I suppose that these are no longer needed since AAD is no longer included in the recordings?